configs/albu_example/mask_rcnn_r50_fpn_1x.py

```diff
@@ -196,7 +196,7 @@
             dict(type='RandomFlip'),
             dict(type='Normalize', **img_norm_cfg),
             dict(type='Pad', size_divisor=32),
-            dict(type='ImageToTensor', keys=['img']),
+            dict(type='DefaultFormatBundle'),
```

Comment:
What is the reason to modify those things? I noticed it breaks the examples using `asyncio`

Reply:
sry, i'm not sure

Reply:
`DefaultFormatBundle` can convert image to [`DataContainer`](https://github.com/open-mmlab/mmcv/blob/c2c9fced2bdd5f80b254723263beca0efb60166c/mmcv/parallel/data_container.py#L19) which can pad multi images to the same shape in a batch.
This pr should update with the upstream.